src/index.ts

```diff
@@ -1,6 +1,6 @@
 import glob from "glob-promise"
 import mkdirp from "mkdirp"
-import path from "path/posix"
+import path from "path"
```

Comment:
we should still use path/posix here if i understand your implementation properly

Reply:
oh maybe not